addons/binding/org.openhab.binding.valloxmv/src/main/java/org/openhab/binding/valloxmv/internal/ValloxMVWebSocket.java

```diff
@@ -257,41 +261,50 @@ public void onBinary(InputStream in) {
 
                 logger.debug("Fan Speed: {}", (bytes[129] & 0xFF));
 
-                int bdFanspeed = getNumber(bytes, 128);
-                int bdFanspeedExtract = getNumber(bytes, 144);
-                int bdFanspeedSupply = getNumber(bytes, 146);
+                int iFanspeed = getNumber(bytes, 128);
+                int iFanspeedExtract = getNumber(bytes, 144);
+                int iFanspeedSupply = getNumber(bytes, 146);
                 BigDecimal bdTempInside = getTemperature(bytes, 130);
                 BigDecimal bdTempExhaust = getTemperature(bytes, 132);
                 BigDecimal bdTempOutside = getTemperature(bytes, 134);
                 BigDecimal bdTempIncomingBeforeHeating = getTemperature(bytes, 136);
                 BigDecimal bdTempIncoming = getTemperature(bytes, 138);
-                int bdHumidity = getNumber(bytes, 166);
+                int iHumidity = getNumber(bytes, 166);
 
-                int bdStateOrig = getNumber(bytes, 214);
-                int bdBoostTimer = getNumber(bytes, 220);
-                int bdFireplaceTimer = getNumber(bytes, 222);
+                int iStateOrig = getNumber(bytes, 214);
+                int iBoostTimer = getNumber(bytes, 220);
+                int iFireplaceTimer = getNumber(bytes, 222);
 
-                int bdCellstate = getNumber(bytes, 228);
-                int bdUptimeYears = getNumber(bytes, 230);
-                int bdUptimeHours = getNumber(bytes, 232);
-                int bdUptimeHoursCurrent = getNumber(bytes, 234);
+                int iCellstate = getNumber(bytes, 228);
+                int iUptimeYears = getNumber(bytes, 230);
+                int iUptimeHours = getNumber(bytes, 232);
+                int iUptimeHoursCurrent = getNumber(bytes, 234);
 
-                int bdExtrFanBalanceBase = getNumber(bytes, 374);
-                int bdSuppFanBalanceBase = getNumber(bytes, 376);
+                int iRemainingTimeForFilter = getNumber(bytes, 236);
+                int iFilterChangedDateDay = getNumber(bytes, 496);
+                int iFilterChangedDateMonth = getNumber(bytes, 498);
+                int iFilterChangedDateYear = getNumber(bytes, 500);
 
-                int bdHomeSpeedSetting = getNumber(bytes, 418);
-                int bdAwaySpeedSetting = getNumber(bytes, 406);
-                int bdBoostSpeedSetting = getNumber(bytes, 430);
+                Calendar cal = Calendar.getInstance();
```

Comment:
`cal` is a very weak name for a variable. Let's use something more identifying here like `cFilterChangedDate`. Wdyt?

Reply:
Sounds good